File: IQ/Classes/clsShoppingListItem.vb


Public Class clsShoppingListItem

    Property QtyPartNo As String
    Property PartNo As String
    Property Quantity As Integer
    Property Product As clsProduct

    Public Sub New(qtyPartNo As String, partNo As String, quantity As Integer, product As clsProduct)

        Me.QtyPartNo = qtyPartNo
        Me.PartNo = partNo
        Me.Quantity = quantity
        Me.Product = product

    End Sub

End Class

File: IQ/Classes/clsShoppingListItem.cs
public class clsShoppingListItem
{

    public string QtyPartNo { get; set; }
    public string PartNo { get; set; }
    public int Quantity { get; set; }
    public clsProduct Product { get; set; }

    public clsShoppingListItem(string qtyPartNo, string partNo, int quantity, clsProduct product)
    {

        this.QtyPartNo = qtyPartNo;
        this.PartNo = partNo;
        this.Quantity = quantity;
        this.Product = product;

    }

}